### ISSUE TITLE
Let's create the comminuty

### ISSUE THREAD

**Issue by ohld** [rarcega/instagram-scraper#526]
Hi! I'm the creator and main developer of https://github.com/instagrambot/instabot

Several largest Instagram-related Open Source projects were taken down recently. My project is the next most popular - I think it will be deleted soon tool. Maybe the same story would happen with your project too.

I think we need to unite our forces and create one big Instagram-relate community to support each other and develop more open-source tools. I already have one: it is located in Telegram. Here is the link: https://t.me/instabotproject. This is a channel (4k members) and a chat (3k members). There are some projects that are already with us, the most important - https://t.me/likemeupbot - 10k active users.

What do you think?

**Comment by VasiliPupkin256** [comment]
What largest project was taken down?

**Comment by ohld** [comment]
Instagram-PHP by mpg25 (the largest ever) and instabot-py (just today)

**Comment by VasiliPupkin256** [comment]
How did you know that? Is it discussed anywhere else? How microsoft reasoned the ban?

**Comment by Aaronzinhoo** [comment]
TOS is not enforceable though against scraping and is completely legal since their content is publicly accessible as stated by a 2018 court decision.... I guess they can still privately enforce it themselves huh? Well we should definitely create some sort of community, to protect the projects we have.

**Comment by Aaronzinhoo** [comment]
@rarcega @AlexNik  I would like to hear your thoughts on this if possible.

**Comment by AlexNik** [comment]
@Aaronzinhoo @ohld I think it's great idea but I don't have any time.
I will read it from time to time... May be)

I use this repo only for downloading live videos for my wife. She buy sometimes "marathons" with live videos but she don't have any time to wath it on air. I try to download it but there was no stuff for it. I found one extension for Chrome but it is has some viruses and then I decided to write a story downloader by my own.

**Comment by 501st-alpha1** [comment]
Throwing in my 2 cents, I'd definitely be in favor of having a backup community that is censorship-resistant.

However, I don't have the bandwidth for keeping up with something like a chat group, so I hope something GitHub-like or at least forum-like could also be set up, so that I can follow topics I'm interested in and ignore the rest.

**Comment by Aaronzinhoo** [comment]
@AlexNik @501st-alpha1  agreed. I believe this decision should ultimately be up to @rarcega. If not him then maybe one of the other consistent contributors. @VasiliPupkin256

**Comment by Aaronzinhoo** [comment]
https://discordapp.com/ This site may be of good use to.

**Comment by aboutqx** [comment]
Thanks.Banning account really disappoint men.
